websites/D/Discord Bots/dist/metadata.json

```diff
@@ -5,18 +5,18 @@
   },
   "service": "Discord Bots",
   "description": {
-    "en": "Discord bots, is listing all discord bots.",
+    "en": "Discord bots is a website that listing discord bots.",
```

Comment:
Change the English description, this isn't grammatically correct. `listing` -> `lists`